internal/api/organization_test.go

```diff
@@ -70,10 +70,16 @@ func TestOrganizationAPI(t *testing.T) {
 				// Default org is already in the database.
 				So(orgs.Result, ShouldHaveLength, 2)
 				orgId := int64(0)
-				So(orgs.Result[1].Name, ShouldEqual, createReq.Name)
-				So(orgs.Result[1].DisplayName, ShouldEqual, createReq.DisplayName)
-				So(orgs.Result[1].CanHaveGateways, ShouldEqual, createReq.CanHaveGateways)
-				orgId = createResp.Id
+				
+				// If the length is not what was expected, then these checks 
+				// will, at best, be checking against some random data, and at
+				// worst, crash the test.
+				if 2 == len(orgs.Result) {
```

Comment:
Note that this is already validated in `So(orgs.Result, ShouldHaveLength, 2)`, the test will not proceed when that assertion is failing.